experimental/dds/tree2/src/class-tree/tree.ts

```diff
@@ -58,6 +61,7 @@ export interface ITree extends IChannel {
 }
 
 /**
+ * Configuration for how to schematize a tree.
```

Comment:
Can we define "schematize" here and/or link to some existing definition elsewhere?

Reply:
```suggestion
 * Configuration for how to {@link ITree.schematize|schematize} a tree.
```